x-pack/plugins/security_solution/public/overview/components/detection_response/translations.ts

```diff
@@ -294,6 +294,6 @@ export const OPEN_IN_ALERTS_TITLE_HOSTNAME = i18n.translate(
 export const OPEN_IN_ALERTS_TITLE_USERNAME = i18n.translate(
   'xpack.securitySolution.detectionResponse.openInAlertsUserName',
   {
-    defaultMessage: 'Username',
+    defaultMessage: 'User name',
```

Comment:
if I'm not mistaken, this is used within the users by alert severity table
![Screenshot 2023-05-05 at 2 01 45 PM](https://user-images.githubusercontent.com/17276605/236546973-1958439e-530c-4bdd-bec1-e141d32ed9f9.png)
if that's the case, I think `Username` is more appropriate. `User name` would refer to the user's full name, but here I believe we're talking about the actual `username`

Reply:
I did not have any particular preference and I wanted uniformity. If you checkout 8-9 lines before this code change... it has names like `Rule name` , `Host name` so I made `Username -> User name`

Reply:
you are right. Please ignore my previous deleted comment. It makes sense.. I will reverse this change. In that case, it will be small case `username` right?

Reply:
I think it's ok to have the first letter capitalized so follow the rest of the columns, `Username` should work